jobs/JGLOBAL_MARINE_BMAT

```diff
@@ -2,17 +2,19 @@
 
 source "${HOMEgfs}/ush/preamble.sh"
 
-if (( 10#${ENSMEM:-0} > 0 )); then
+# source config.base, config.ocnanal and config.marinebmat
+# and pass marinebmat to ${machine}.env
+source "${HOMEgfs}/ush/jjob_header.sh" -e "marinebmat" -c "base ocnanal marinebmat"
+
+
+if (( 10#${NMEM_ENS:-0} > 0 )); then
   export DATAjob="${DATAROOT}/${RUN}marinebmat.${PDY:-}${cyc}"
-  export DATA="${DATAjob}/${jobid}"
+  export DATA=${DATA:-${DATAROOT}/${jobid:?}}
   # Create the directory to hold ensemble perturbations
   export DATAenspert="${DATAjob}/enspert"
   if [[ ! -d "${DATAenspert}" ]]; then mkdir -p "${DATAenspert}"; fi
 fi
```

Comment:
Why is this being moved after the jjob_header call? jjob_header sets up `$DATA`, so not sure why the order is changing here (I also don't think `DATA` needs to be defined at all, since this is what jjob_header uses anyway).